build.sbt

```diff
@@ -17,9 +17,14 @@ val MacOS = "macos-latest"
 ThisBuild / githubWorkflowOSes := Seq(PrimaryOS, MacOS)
 
 val PrimaryJava = JavaSpec.temurin("8")
-val LTSJava = JavaSpec.temurin("17")
+val LTSJava = JavaSpec.temurin("21")
 ThisBuild / githubWorkflowJavaVersions := Seq(PrimaryJava, LTSJava)
 
+// MacOS runners do not have temurin@8
+ThisBuild / githubWorkflowBuildMatrixExclusions := Seq(
+  MatrixExclude(Map("os" -> MacOS, "java" -> JavaSpec.temurin("8").render))
```

Comment:
This could use the `PrimaryJava` constant, but we really only need to drop what MacOS has dropped.  Whenever `PrimaryJava` progresses to 11, this can be removed... unless MacOS has already dropped that, too.

Reply:
Would it be reasonable to bump `PrimaryJava` to 11 now? I don't have a lot of context but this is for new projects, and 8 seemed to be accelerating on it's way out

Reply:
I think we could bump to 11, but that can also happen in a follow up

Reply:
I asked Andrew about this and he proposed doing this bump in a follow-up just so it's more explicit